hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/hevm/HederaEvmBlocks.java

```diff
@@ -48,7 +48,7 @@ public interface HederaEvmBlocks {
      */
     BlockValues blockValuesOf(long gasLimit);
 
-    static Hash besuHashFrom(@NonNull final com.swirlds.common.crypto.Hash hash) {
+    private static Hash besuHashFrom(@NonNull final com.swirlds.common.crypto.Hash hash) {
```

Comment:
Is that change needed for the PR?

Reply:
Not necessarily, but I wanted the dependency check to pass. And this was also an example of what you can detect with the check. See these comments: https://github.com/hashgraph/hedera-services/pull/7330#discussion_r1258458493



Note: strictly speaking the dependency check should not have complained here. It should **not** consider package-private things to be public API in a Module context. Because there, a package is always only accessible within the module. This is different to classpath-based Java. I already took a note to provide a fix for this in the analysis plugin.